tests/HashMapProperties.hs

```diff
@@ -6,6 +6,7 @@
 module Main (main) where
 
 import Control.Monad ( guard )
+import Control.Monad.State ( execState, get, put )
```

Comment:
wont `transformers` (i.e. `Control.Monad.Trans.State`) be enough? (though it's in tests, so probably not a big deal)